packages/react-tooltip/src/components/Tooltip/Tooltip.test.tsx

```diff
@@ -1,16 +1,25 @@
 import * as React from 'react';
 import { Tooltip } from './Tooltip';
-import * as renderer from 'react-test-renderer';
-import { ReactWrapper } from 'enzyme';
 import { isConformant } from '../../common/isConformant';
+import { render, RenderResult } from '@testing-library/react';
 
 describe('Tooltip', () => {
   isConformant({
     Component: Tooltip,
     displayName: 'Tooltip',
+    requiredProps: { content: 'Example', children: <button /> },
+    disabledTests: [
+      // Tooltip renders into a Portal, which confuses these tests
+      'component-handles-ref',
+      'component-has-root-ref',
+      'component-handles-classname',
+      'as-renders-fc',
+      'as-passes-as-value',
+      'as-renders-html',
+    ],
   });
 
-  let wrapper: ReactWrapper | undefined;
+  let wrapper: RenderResult | undefined;
```

Comment:
nit: do you need this to persist across all tests ? can each test just manage its own render result ?

Reply:
Honestly I didn't give it much thought; this code is based on the template. It looks like this is done in order to call `unmount()` after each test. Do you know if that is necessary?
```ts
  afterEach(() => {
    if (wrapper) {
      wrapper.unmount();
      wrapper = undefined;
    }
  });
```

Reply:
This is actually a specific thing about the previous test framework `enzyme` which does some magical things during component mount.

The `testing-library` render is light weight and based off the native react test renderer, so there's no reason to have any references persist across tests, it makes it more risky that tests start depending on each other